install_config/upgrading/manual_upgrades.adoc

```diff
@@ -215,7 +280,6 @@ endif::[]
 +
 ----
 # oadm policy reconcile-cluster-role-bindings \
-    --exclude-groups=system:authenticated \
     --exclude-groups=system:authenticated:oauth \
```

Comment:
@liggitt After a 3.2.z upgrade, I wasn't able to do builds for an existing user until I re-ran `reconcile-cluster-role-bindings` w/ `--exclude-groups=system:authenticated` dropped, as seen above.

I see this thread on the subject: http://lists.openshift.redhat.com/openshift-archives/users/2016-June/msg00013.html

Do you suggest dropping `--exclude-groups=system:authenticated` here, or leaving it and providing better instruction, like maybe linking to https://docs.openshift.com/enterprise/latest/admin_guide/securing_builds.html#restricting-build-strategies-to-a-user-globally?